### ISSUE TITLE
Task 3 is not correct

### ISSUE THREAD

**Issue by jablka** [dpertsin/codergames2024#1]
Yesterday I finnished Task 3 (for myself, I know it's late).
And I wanted to compare my results with someone else's, so I found your code.
But I noticed that your code gives in fact wrong results!
If you check with the sample imput data:
4
2 1 1 3 3 2
1 2 1 3 1 0

The output should be: [-1, 2, 1, 3, 1, -2]

(But your code gives: [-1, 1, 0, 0, 2, 2])

I'm just letting you know.
Thanks

**Comment by jablka** [comment]
UPDATE:
I just found that on the website they revealed the correct answers (and in fact they say Task3 can have many possible answers).
But anyway, you might double check...

**Comment by dpertsin** [comment]
Yes, you are right, I will check it in the future. Thank you for letting me know.

**Comment by jablka** [comment]
UPDATE,
I suspect even official solution contain errors (inconsistencies)...
I just emailed them about it.
